VultisigApp/VultisigApp/Views/Settings/SettingsBiometryView.swift

```diff
@@ -28,6 +28,19 @@ struct SettingsBiometryView: View {
     }
 
     var main: some View {
+        VStack {
+            headerMac
```

Comment:
You should only apply `headerMac` when it is running on a Mac , Not IOS

<img width="879" alt="image" src="https://github.com/user-attachments/assets/e79c2db9-e29a-4ce5-81ae-784222f7c59b">